integration_tests/Cargo.toml

```diff
@@ -575,6 +575,22 @@ path = "../rusoto/services/workspaces"
 optional = true
 path = "../rusoto/services/xray"
 
+[dependencies.rusoto_amplify]
```

Comment:
I've been trying to keep this file alphabetized. Would the possible refactoring of this file include that in a followup PR?

Reply:
Absolutely. I was even going to suggest it. I kind of feel like this file itself could be generated but ill do the mvp thing first

Reply:
I ran just gave https://github.com/tbrand/cargo-tomlfmt a split and it wasn't exactly what I wanted but it was an improvement. I'll open an separate issue to follow up on this